pkgs/top-level/python-packages.nix

```diff
@@ -15287,15 +15287,15 @@ in modules // {
   });
 
   psycopg2 = buildPythonPackage rec {
-    name = "psycopg2-2.5.4";
+    name = "psycopg2-2.6.1";
     disabled = isPyPy;
 
-    # error: invalid command 'test'
+    # error: SyntaxError: invalid syntax
     doCheck = false;
```

Comment:
Thanks for the update. It would be good if the tests work. Apparently there is a `SyntaxError` now then. Do you know whether it is in the tests or in the library?

Anyway, I think we could disable the tests for this because they require a database.